<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addBookButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>751</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Book</string>
    </property>
   </widget>
   <widget class="QLabel" name="heading">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>216</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Qt Reading List</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="isbnField">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>999-9-99999-999-9</string>
    </property>
    <property name="text">
     <string>----</string>
    </property>
    <property name="placeholderText">
     <string>ISBN</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
   </widget>
   <widget class="QLabel" name="authorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
   </widget>
   <widget class="QLabel" name="publisherLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Publisher:</string>
    </property>
   </widget>
   <widget class="QLabel" name="isbnLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ISBN Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleField">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>60</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="authorField">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="publisherField">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="readingList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>751</width>
      <height>191</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
font: 700 9pt &quot;Segoe UI&quot;;

</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
